index.html

```diff
@@ -3514,6 +3514,58 @@ <h3>Reserved Extension Points</h3>
         </p>
 
       </section>
+
+      <section class="normative">
+        <h3>Ecosystem Compatibility</h3>
+
+        <p>
+There are a number of digital credential formats that do not natively use the
+data model provided in this document, but are aligned with a number of concepts
+in this specification. At the time of publication, examples of these digital
+credential formats include pure JSON Web Tokens (JWTs), CBOR Web Tokens (CWTs),
+ISO-18013-5 (mDLs), and Authentic Chained Data Containers (ACDCs).
+        </p>
+
+        <p>
+If conceptually aligned digital credential formats can be transformed to the
+data model according to the rules provided in this section, they are considered
+<em>"compatible with the Verifiable Credentials ecosystem"</em>, and the result
+of the transformation, which is a <a>conforming document</a>, is a
```

Comment:
a conforming document is either a `vc+ld+json` or a `vp+ld+json`... the term conforming document can stay, but using the media types here, will make the sentences below easier to understand because you can see the need for the statements based on the `+ld+json` suffix.

Reply:
@OR13 I have made it explicit which media types are allowed for a `conforming document` in bd9aab2a283bd45789984fe2fc528c696ebe2aac.